docs/release/trg-0/trg-1-09.md

```diff
@@ -0,0 +1,43 @@
+---
+title: TRG 1.09 - UX consistency Style Guideline for UI
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status                                    | Created        | Post-History                                       |
+|-------------------------------------------|-----------------|----------------------------------------------------|
+| Draft - MISSING CLOSED INFORMATION      | 10-Jan-2024     | Transfered from Release Acceptance Criteria into TRG|
+
+## Why
+
+Ensuring UX consistency through a style guideline is essential for a unified and user-friendly experience across front end modules.
+
+## Description
+
+- UX consistency is mandatory for FrontEnd modules.
+- User Interfaces should align with the Catena-X Style Guidelines.
+
+> User interface style review has been executed.
+> Review feedback (if existing) got incorporated.
+> All findings are assessed.
+> All findings (high/very high) are fixed or cleaned up (evidence by re-review).
+> Approval of the application CX Style conformity is available (given by the review owner).
+
+## Best Practice
+
```

Comment:
Make use of the public available shared component library (react supported) to easily develop applications that are in line with the Tractus-X style guidelines.